testing/calibration/src/lib.rs

```diff
@@ -24,7 +24,7 @@ use num_traits::Zero;
 use serde::Serialize;
 
 pub const WASM_COMPILED_PATH: &str =
-    "../../target/debug/wbuild/fil_gas_calibration_actor/fil_gas_calibration_actor.compact.wasm";
+    "../../target/release/wbuild/fil_gas_calibration_actor/fil_gas_calibration_actor.compact.wasm";
```

Comment:
This was the culprit for some major headaches. We run the harness in `--release` profile, which means that we recompile contract changes under the `release` profile. Unfortunately the calibration harness was picking the actor up from the `debug` profile, which usually contains a stale version.

@Stebalien -- this might have or might have not affected your readings.

Reply:
It shouldn't matter. I was timing FVM operations, not things within the actors.

Reply:
Correct, it should not affect the readings. However, in my case this ended up picking stale actor code more than once, leading to wrong conclusions. So wanted to give you a heads-up just in case.

Reply:
Oh.... Damn, I see. I think rust-analyzer's background "let me recompile everything for you and melt a hole in your lap" helped there.